hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java

```diff
@@ -87,6 +87,10 @@
   private String[] args;
   private volatile AtomicBoolean isStopped = new AtomicBoolean(false);
 
+  public HddsDatanodeService(){
+    this(false, new String[] {});
```

Comment:
Calling the other constructor is unnecessary, the default values for `printBanner` (`false`) and `args` (`null`) are fine.

On the other hand, can you please add a comment that this constructor is required for `ServicePlugin` implementation?  This would help later to avoid accidentally removing it again.

Reply:
@adoroszlai ,thanks for review , files updated.

Reply:
Thanks.